src/main/java/uk/gov/ch/oauth/IIdentityProvider.java

```diff
@@ -49,4 +51,6 @@
     String getRedirectUriPage();
     
     String getAccountLocalUrl();
+    
+    String getGrantType();
```

Comment:
Does grant type get used anywhere outside of the identity provider? If not, drop it from the interface.

Reply:
Much like the other fields they are only used within the Identity provider.

Reply:
For future proofing, we might handle different grant types which is why I put it in the interface.

Reply:
It would still only be used internally, so please drop it out, for now.

Reply:
The same could be said for some of the other fields, I don't think it makes a difference.